src/plots/Mesh/QvisMeshPlotWindow.C

```diff
@@ -205,9 +205,12 @@ QvisMeshPlotWindow::CreateWindowContents()
     rb->setChecked(true);
     meshColorButtons->addButton(rb, 0);
     colorLayout->addWidget(rb, 0, 1);
+    rb = new QRadioButton(tr("Random"), central);
+    meshColorButtons->addButton(rb, 2);
```

Comment:
Just a nit, but if 'Random' comes before 'Custom', shouldn't its' id be smaller (eg 1), and Custom's id be 2?

Reply:
Well, its third in the enum but second in the GUI widget set (foreground, random, custom). The reason being is that custom *must* go at the end because there is the color selector box that follows it. 
![Untitled](https://user-images.githubusercontent.com/5720676/58644950-37e6a800-82b7-11e9-972f-76a62720c255.png)

Reply:
Yeah, I understood that about the gui. Also, they currently need to be the id you've chosen because of the order they appear in the atts enum. You can ignore me.